### DISCUSSION TITLE
LDAP support

### DISCUSSION THREAD

**Discussion by RatCornu** [stoatchat/discussions#726] [Feature Suggestions]
Hello! I'm verry interested in this project but I would need an external auth to be able to use it with my friends on a server. I saw that Oauth2 is in the Roadmap which is great, but I didn't see LDAP support. Would this be something considerated ?

For example, https://github.com/inejge/ldap3 seems to be an actively maintained LDAP client library for Rust which could be used.

**Comment by notjiatan** [comment]
bump

**Comment by locaIhost** [comment]
bump
